lxd/instances_post.go

```diff
@@ -11,7 +11,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/dustinkirkland/golang-petname"
+	petname "github.com/dustinkirkland/golang-petname"
```

Comment:
I'd rather not carry this, just as we don't write `lxd "github.com/lxc/lxd/client"` everywhere.

Reply:
Hrm, ok, seems like its been encouraged with the new gopls server in vscode, as it just started doing that the other day without me changing any settings. I'll see if I can turn it off otherwise this is going to keep happening automatically.

Reply:
I believe @freeekanayaka had this similarly show up before with whatever Emacs magic he's using.

If we do end up having to go down that route (the fact that vet and lint don't complain suggests we don't have to), then we should do it throughout the entire codebase in one shot and add static analysis to catch regressions, but I personally don't like it and prefer to keep this syntax strictly used for cases where we rename a package on our side, not to show their default name.

Reply:
OK I've managed to coax vscode into stopping messing with go imports using:

```
    "go.useLanguageServer": true,
    "[go]": {
        "editor.formatOnSave": true,
        "editor.codeActionsOnSave": {
            "source.organizeImports": false,
        }
    },
```